### ISSUE TITLE
pdf.bernoulli isn't vectorised

### ISSUE THREAD

**Issue by 1danjordan** [alexpghayes/distributions#12]
See below:

```r
b <- bernoulli()
x <- c(0,0,0,1)
pdf(b, x)

#> [1] 0.5
#> Warning message:
#> In if (x == 0) 1 - d$p else d$p :
#>   the condition has length > 1 and only the first element will be used
```

Simple enough fix is to use `dbinom` with `size = 1` instead. I can include this fix in my existing pull request.

**Comment by alexpghayes** [comment]
That would be great!

**Comment by 1danjordan** [comment]
Fixed in #5
